chatmaild/src/chatmaild/config.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, inipath, params):
         self.passthrough_recipients = params["passthrough_recipients"].split()
         self.filtermail_smtp_port = int(params["filtermail_smtp_port"])
         self.postfix_reinject_port = int(params["postfix_reinject_port"])
+        self.disable_ipv6 = params.get("disable_ipv6")
```

Comment:
```suggestion
        self.disable_ipv6 = True if params.get("disable_ipv6") == "True" else False
```

Let's make a boolean out of it?

Reply:
Originaly this was a boolean like that - I changed it back to a string because of the weird string related issue above